File: python/Human.py
class Human:

    def __init__(self, NIK = "", nama = "", jeniskelamin = ""):
        self.__NIK = NIK
        self.__nama = nama
        self.__jeniskelamin = jeniskelamin

    def getNIK(self):
        return self.__NIK

    def setNIK(self, NIK):
        self.__NIK = NIK

    def getnama(self):
        return self.__nama

    def setnama(self, nama):
        self.__nama = nama

    def getjeniskelamin(self):
        return self.__jeniskelamin

    def setjeniskelamin(self, jeniskelamin):
        self.__jeniskelamin = jeniskelamin

File: cpp/human.cpp
#include <iostream>
#include <string>

using namespace std;

class human
{

private:
    string nama, nik, gender;

    // Konstruktor
public:
    human()
    {
        this->nama = "";
        this->nik = "";
        this->gender = "";
    }

    human(string nama, string nik, string gender)
    {
        // konstruktor dengan parameter
        this->nama = nama;
        this->nik = nik;
        this->gender = gender;
    }

    // setter getter

    // SETTER

    void setnama(string nama)
    {
        this->nama = nama;
    }

    void setnik(string nik)
    {
        this->nik = nik;
    }

    void setgender(string gender)
    {
        this->gender = gender;
    }

    // GETTER

    string getnama()
    {
        return this->nama;
    }

    string getnik()
    {
        return this->nik;
    }

    string getgender()
    {
        return this->gender;
    }

    ~human()
    {
    }
};